ivy/functional/frontends/tensorflow/math.py

```diff
@@ -612,6 +614,12 @@ def scalar_mul(scalar, x, name="scalar_mul"):
     return ivy.multiply(x, scalar).astype(x.dtype)
 
 
+@to_ivy_arrays_and_back
+def segment_sum(data, segment_ids, name="segment_sum"):
+    num_segments = np.array(np.max(segment_ids) + 1)
```

Comment:
Why are we using numpy here, we can use ivy, right?

Reply:
Yes, i should have used ivy here, i just realized and i am trying to change from numpy to ivy but i am failing some test cases for tensorflow when using ivy instead of numpy because of the item method. Do ivy.array have any item() method when using with tensorflow? i will update as soon as i get my test cases completed.